chapters/hlsl.adoc

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021 The Khronos Group, Inc.
+// Copyright 2021 Sascha Willems
+// SPDX-License-Identifier: CC-BY-4.0
+
+ifndef::chapters[:chapters:]
+
+[[hlsl-in-vulkan]]
+= HLSL in Vulkan
```

Comment:
would point a link here in the "what is SPIR-V" chapter!

Reply:
I added a link to the first occurrence of HLSL, but it doesn't seem to work.

Code looks like this:

```
xref:{chapters}hlsl.adoc[HLSL]
```

But clicking on it results in an URl ending with `chapters/%7Bchapters%7Dhlsl.adoc` that doesn't work. Any idea? Is the link format wrong? That same format works fine in my new chapter.

Reply:
awww, the spir-v chapter is missing

```
ifndef::chapters[:chapters:]
```
let me add this for any other missing chapters to

So just leave (or rebase if you want) because when the merge happens from this PR it will all just work